app/util/k8s/terminate_cluster.py

```diff
@@ -373,6 +443,7 @@ def terminate_cluster(cluster_name, aws_region=None):
     # Delete the nodegroup and cluster in the specified region
     delete_nodegroup(aws_region, cluster_name)
     delete_cluster(aws_region, cluster_name)
+    delete_hosted_zone_record_if_exists(aws_region, cluster_name)
```

Comment:
Is it possible to get info about hosted zone for cluster from it's configuration?